opm/simulators/flow/DamarisWriter.hpp

```diff
@@ -61,13 +61,14 @@ namespace Opm {
 
 namespace DamarisOutput {
 
-int endIteration(int rank);
-int setParameter(const char* field, int rank, int value);
-int setPosition(const char* field, int rank, int64_t pos);
-int write(const char* field, int rank, const void* data);
+int endIteration();
+int setParameter(const char* field, int value);
+int setPosition(const char* field, int64_t pos);
+int write(const char* field, const void* data);
```

Comment:
Tangential comment, not related to the topic of this PR: Can we make these `const char*` parameters be `std::string_view` instead&ndash;perhaps in follow-up work?